packages/pds/src/account-manager/helpers/account.ts

```diff
@@ -62,16 +72,21 @@ export const registerActor = async (
   opts: {
     did: string
     handle: string
+    deactivated?: boolean
   },
 ) => {
-  const { did, handle } = opts
+  const { did, handle, deactivated } = opts
   const [registered] = await db.executeWithRetry(
     db.db
       .insertInto('actor')
       .values({
         did,
         handle,
         createdAt: new Date().toISOString(),
+        deactivatedAt: deactivated ? new Date().toISOString() : null,
+        deleteAfter: deactivated
+          ? new Date(Date.now() + 3 * DAY).toISOString()
+          : null,
```

Comment:
It's total nit but it would be nice for both of these values to be based on the same date.  E.g.
```ts
const now = new Date()
// ...
  deactivatedAt: deactivated ? now.toISOString() : null,
  deleteAfter: deactivated
    ? new Date(now.getTime() + 3 * DAY).toISOString()
    : null,
```